src/test/java/ladder/domain/LineTest.java

```diff
@@ -0,0 +1,25 @@
+package ladder.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LineTest {
+
+    @Test
+    @DisplayName("사다리 라인을 생성한다.")
+    void createLine() {
+        assertThat(new Line(3, () -> true).getList()).isEqualTo(List.of(true, false, true));
+        assertThat(new Line(3, () -> false).getList()).isEqualTo(List.of(false, false, false));
```

Comment:
```suggestion
        assertThat(new Line(3, () -> false)).isEqualTo(new Line(List.of(false, false, false));
```
게터 없이도
Line 값 끼리 비교해 볼 수 있게 만들어봐도 좋겠네요 👍 🚀